pages/profile.php

```diff
@@ -21,7 +23,64 @@
         </tr>
         <tr>
             <th>
-                <?php esc_html_e('PGP email subject lines', 'wp-pgp-encrypted-emails');?>
+                <label for="<?php print esc_attr(self::$meta_key_smime);?>">
+                    <?php esc_html_e('S/MIME Public Certificate', 'wp-pgp-encrypted-emails');?>
+                </label>
+            </th>
+            <td>
+                <textarea
+                    id="<?php print esc_attr(self::$meta_key_smime);?>"
+                    name="<?php print esc_attr(self::$meta_key_smime)?>"
+                    class="large-text code"
+                    rows="5"
+                    ><?php print esc_textarea($profileuser->{self::$meta_key_smime});?></textarea>
+                <p class="description">
+                    <?php print sprintf(
+                        esc_html__('Paste your S/MIME public certificate here to have WordPress encrypt emails it sends you. Leave this blank if you do not want to get or know how to decrypt encrypted emails.', 'wp-pgp-encrypted-emails')
+                    );?>
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <th>
+                <?php esc_html_e('Use PGP or S/MIME?', 'wp-pgp-encrypted-emails');?>
+            </th>
+            <td>
+                <label>
+                    <input type="radio"
+                        id="<?php print esc_attr(self::$meta_key_encryption_type);?>-0"
+                        name="<?php print esc_attr(self::$meta_key_encryption_type);?>"
+                        <?php checked($profileuser->{self::$meta_key_encryption_type},0);?>
+                        value="0"
+                    />
+                    <?php esc_html_e('No encryption', 'wp-pgp-encrypted-emails');?>
+                </label><br/>
+                <label>
+                    <input type="radio"
+                        id="<?php print esc_attr(self::$meta_key_encryption_type);?>-1"
+                        name="<?php print esc_attr(self::$meta_key_encryption_type);?>"
+                        <?php checked($profileuser->{self::$meta_key_encryption_type},1);?>
+                        value="1"
```

Comment:
I don't like these magic numbers. If we're going to use an encryption toggle (and I'm not sure that we should…?) then we should at least be explicit in our own code about what the toggle actually is used for. There's no space constraint for this setting, so we should use string names for these values such as `pgp` or `smime`, for example, not just `1` or `2`.